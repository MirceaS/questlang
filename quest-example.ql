World
  Location Forest
  Location Desert
  NPC Player at Forest
  NPC Wolf at Forest
  Item Sword at Desert
  Wolf Vulnerable to (Sword)

Subquest RoundTrip (location, item)
  let firstLoc = getloc Player
  goto location
  get item
  goto firstLoc

Quest
  run RoundTrip (Location Desert, Item Sword)
  kill Wolf