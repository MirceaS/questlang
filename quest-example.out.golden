Validating...
Quest was validated successfully!
